go/protocol/keybase1/extras_test.go

```diff
@@ -33,6 +33,30 @@ func TestTime(t *testing.T) {
 	}
 }
 
+func TestTimeConversions(t *testing.T) {
+	times := []time.Time{
+		time.Time{},
+		time.Now(),
+		time.Now().AddDate(1000, 0, 0),  // in a thousand years
+		time.Now().AddDate(10000, 0, 0), // in 10 thousand years
+		time.Now().AddDate(-3000, 0, 0), // 3 thousand years ago
+	}
+
```

Comment:
Can we add tests that specifically test the 0 value?
```
	require.Equal(t, UnixTime(0), ToUnixTime(time.Time{}))
	require.Equal(t, Time(0), ToTime(time.Time{}))

	require.Equal(t, time.Time{}, FromUnixTime(UnixTime(0)))
	require.Equal(t, time.Time{}, FromTime(Time(0)))
```

Reply:
I think time.Time{} in the test already covers this case, but I'll put a more obvious test

Reply:
correct me if I'm wrong, but reverse conversion test would also succeed if time.Time=0 was converted to some non integer=0  constant and then happened to convert back successfully, which is not exactly what we want

Reply:
Oh yes, that's true. Thanks :)